mesonbuild/compilers/compilers.py

```diff
@@ -112,6 +112,12 @@ def is_library(fname):
                           'debugoptimized': ['-O2', '-g'],
                           'release': ['-O3'],
                           'minsize': ['-Os', '-g']}
+arm_buildtype_args = {'plain': [],
+                      'debug': ['-O0', '-g'],
+                      'debugoptimized': ['-O2', '-g'],
+                      'release': ['-O2'],
+                      'minsize': ['-Os', '-g'],
+                     }
```

Comment:
__[Flake8]__

[E124] closing bracket does not match visual indentation


<!-- Comment by [SideCI](https://sideci.com) -->

Reply:
Agreed. Will submit new patch change to reflect these changes